### DISCUSSION TITLE
Release Created on Commit Hash Prior to Tag

### DISCUSSION THREAD

**Discussion by davidlday** [ncipollo/release-action#290] [Q&A]
I'm clearly doing something wrong here. I'm using `mathieudutour/github-tag-action`, and then my next step is using `ncipollo/release-action`. But when I check the release, it shows one commit since the release, which is the prior tag action.

Example workflow: https://github.com/prosegrinder/python-syllables/actions/runs/3669892651/jobs/6204054250
Example release: https://github.com/prosegrinder/python-syllables/releases/tag/v1.0.4

I'm not passing in `commit`, and feel like I should be passing in the new tag, but the docs only specify passing in a branch or a commit hash. Should I be passing in `main` for it to pick up the pushed tag? Can I pass the tag in?

Thanks in advance!

Edit: forgot to mention, I'm using annotated tags in the tag action.

**Comment by EmelianLisovski-eaton** [comment]
I've seen similar issues using annotated tags. From my perspective it seems to be isolated to individual projects / use cases with git describe and falls outside of the scope of this Action. I would recommend checking that git describe or similar is not used elsewhere in a project and is accidentally interfering with the tag passed to your workflow. When troubleshooting this error I've found it helpful to verify annotated tags locally, isolated from any workflows. Hope this helps
